src/components/Text.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+function Text({ size, color, children}) {
```

Comment:
`const { theme } = useContext(ThemeContext);` should be used here to control the text color